### DISCUSSION TITLE
1970/01/01/hello-world

### DISCUSSION THREAD

**Discussion by giscus** [harukin0731/xprocess-info-blog#1] [Announcements]
# 1970/01/01/hello-world

Welcome to Hexo ! This is your very first post. Check documentation  for more info. If you get any problems when using Hexo, you can find the answer in troubleshooting  or you can ask me on GitHub . Q

https://770b4bd1.xprocess-info-blog.pages.dev/1970/01/01/hello-world

**Comment by harukin0731** [comment]
testarea~ :)

**Reply by harukin0731** [reply]
# test
